java/src/jmri/jmrix/srcp/SRCPThrottleManager.java

```diff
@@ -81,8 +82,8 @@ static boolean isLongAddress(int num) {
     }
 
     @Override
-    public int supportedSpeedModes() {
-        return (DccThrottle.SpeedStepMode128 | DccThrottle.SpeedStepMode28);
+    public EnumSet<SpeedStepMode> supportedSpeedModes() {
+        return EnumSet.of(SpeedStepMode.NMRA_DCC_128, SpeedStepMode.NMRA_DCC_28);
```

Comment:
This wasn’t right before or after the changes.  14 and 27 step mode are also supported.

Reply:
lol. Fixed.